helix-rest/src/test/java/org/apache/helix/rest/server/TestInstancesAccessor.java

```diff
@@ -46,8 +46,34 @@
   private final static String CLUSTER_NAME = "TestCluster_0";
 
   @Test
-  public void testInstancesStoppable_zoneBased() throws IOException {
+  public void testInstancesStoppable_zoneBased() throws Exception {
     System.out.println("Start test :" + TestHelper.getTestMethodName());
+    // Make sure that cluster config exists
```

Comment:
Shall we put the wait to the AbstractTestClass? Since the cluster and instances were created there. This check does not make sense in this specific test class scenario.

Reply:
Done.